modules/cert-manager-uninstall-console.adoc

```diff
@@ -13,10 +13,53 @@ You can uninstall the {cert-manager-operator} by using the web console.
 * You have access to the cluster with `cluster-admin` privileges.
 * You have access to the {product-title} web console.
 * The {cert-manager-operator} is installed.
+* You have a backup of cluster issuers and certificates.
 // TODO: Any other prereqs, like removing anything that is using it?
 
 .Procedure
 
+. Get a list of issuers by running the following command:
++
+[source,terminal]
+----
+$ oc get issuers.cert-manager.io -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE      NAME                               READY   AGE
+cert-manager   letsencrypt-issuer                 True    19h
+----
+
+. Get a list of cluster issuers by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterissuers.cert-manager.io
+----
++
+.Example output
+[source,terminal]
+----
+NAME                        READY   AGE
+letsencrypt-clusterissuer   True    19h
+----
+
+. You can get a list of certificates by running the following command:
++
+[source,terminal]
+----
+$ oc get certificates.cert-manager.io -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE  NAME                    READY  SECRET                  AGE
+kafka      my-cluster-certificate  True   my-cluster-certificate  18h
```

Comment:
Do you mind changing the namespace to something else, aside of `kafka`? As seeing kafka could be confusing to some users. Any other namespace should work.